Rakefile

```diff
@@ -67,67 +67,55 @@ RSpec::Core::RakeTask.new("spec:tools") do |t|
-    conf.obs_api = "https://api.opensuse.org"
-    conf.obs_project = "systemsmanagement:machinery"
-    conf.package_name = "machinery"
-    conf.obs_target = "openSUSE_13.2"
-
-    #lets ignore license check for now
-    conf.skip_license_check << /.*/
-  end
-end
-
-def configure_machinery_helper
-  Packaging.configuration do |conf|
-    conf.obs_api = "https://api.opensuse.org"
-    conf.obs_project = "systemsmanagement:machinery:helper"
-    conf.package_name = "machinery-helper-x86_64"
-    conf.obs_target = "SLE_12"
-
-    #lets ignore license check for now
-    conf.skip_license_check << /.*/
-  end
+Packaging.configuration do |conf|
+  conf.obs_api = "https://api.opensuse.org"
+  conf.obs_project = "systemsmanagement:machinery"
+  conf.package_name = "machinery"
+  conf.obs_target = "openSUSE_13.2"
+  conf.version = Machinery::VERSION
+
+  #lets ignore license check for now
+  conf.skip_license_check << /.*/
 end
 
 namespace :man_pages do
   desc 'Build man page(s)'
   task :build do
-    if Dir.exist?("man")
-      puts "  Building man pages"
-      manpage = ""
-      Inspector.all_scopes.each do |scope|
-        manpage += "* #{scope}\n\n"
-        manpage += File.read("plugins/#{scope}/#{scope}.md")
-        manpage += "\n"
-      end
+    puts "  Building man pages"
+    manpage = ""
+    Inspector.all_scopes.each do |scope|
+      manpage += "* #{scope}\n\n"
+      manpage += File.read("plugins/#{scope}/#{scope}.md")
       manpage += "\n"
-      File.write("man/generated/machinery_main_scopes.1.md", manpage)
-
-      system <<-EOF
-        cat man/machinery_main_general.1.md man/generated/machinery_main_scopes.1.md \
-          man/machinery_main_usecases.1.md man/machinery-*.1.md \
-          man/machinery_footer.1.md  > man/generated/machinery.1.md
-      EOF
-      system "sed -i '/<!--.*-->/d' man/generated/machinery.1.md"
-      system "ronn man/generated/machinery.1.md"
-      system "gzip -f man/generated/*.1"
-      # Build man page for website (manual.html)
-      system "ronn -f man/generated/machinery.1.md"
-      system "man/generate_man"
     end
+    manpage += "\n"
+    File.write("man/generated/machinery_main_scopes.1.md", manpage)
+
+    system <<-EOF
+      cat man/machinery_main_general.1.md man/generated/machinery_main_scopes.1.md \
+        man/machinery_main_usecases.1.md man/machinery-*.1.md \
+        man/machinery_footer.1.md  > man/generated/machinery.1.md
+    EOF
+    system "sed -i '/<!--.*-->/d' man/generated/machinery.1.md"
+    system "ronn man/generated/machinery.1.md"
+    system "gzip -f man/generated/*.1"
+    # Build man page for website (manual.html)
+    system "ronn -f man/generated/machinery.1.md"
+    system "man/generate_man"
   end
 end
 
+# Disable packaging_tasks' tarball task. We package a gem, so we don't have to
+# put the sources into IBS. Instead we build the gem in the tarball task
+Rake::Task[:tarball].clear
+task :tarball => ["man_pages:build"] do
```

Comment:
Because it is part of packaging tasks and normally nearly all obs packages work with tarballs.